# These three source files cause build failures on i686 and armv6l.
# WITH_CR32S enables soxr's CR32S SIMD kernels, but is always OFF.

--- a/src/au3wrap/thirdparty/soxr/CMakeLists.txt
+++ b/src/au3wrap/thirdparty/soxr/CMakeLists.txt
@@ -8,17 +8,14 @@
 list( APPEND SOURCES
    PRIVATE
       ${TARGET_ROOT}/src/cr.c
       ${TARGET_ROOT}/src/cr32.c
-      ${TARGET_ROOT}/src/cr32s.c
       ${TARGET_ROOT}/src/cr64.c
       ${TARGET_ROOT}/src/data-io.c
       ${TARGET_ROOT}/src/dbesi0.c
       ${TARGET_ROOT}/src/fft4g.c
       ${TARGET_ROOT}/src/fft4g32.c
       ${TARGET_ROOT}/src/fft4g64.c
       ${TARGET_ROOT}/src/filter.c
-      ${TARGET_ROOT}/src/pffft32s.c
       ${TARGET_ROOT}/src/soxr.c
-      ${TARGET_ROOT}/src/util32s.c
       ${TARGET_ROOT}/src/vr32.c
 )
